pkg/cortex/serve/start/batch.py

```diff
@@ -269,8 +271,7 @@ def handle_on_job_complete(message):
                 should_run_on_job_complete = True
             time.sleep(10)  # verify that the queue is empty one more time
     except:
-        logger.exception("failed to handle on_job_complete")
-        raise
+        raise CortexException("failed to handle on_job_complete")
```

Comment:
We should also add the initial stack trace with `raise CortexException("..") from e`